client/main.js

```diff
@@ -37,3 +38,13 @@ AutoForm.addHooks(['partyForm'],{
     FlowRouter.go('/party/' + this.docId);
   }
 });
+
+Meteor.subscribe('profiles');
+
+Meteor.setInterval(() => {
```

Comment:
So, question for you: why are we doing this check every 100ms? Is there a hook we could use on login attempts where we could add this logic just to check as needed?